packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/ChatbotMessage/ChatbotMessage.md

```diff
@@ -85,3 +85,11 @@ The API for a source requires a link at minimum, but we strongly recommend provi
 ```js file="./MessageWithSources.tsx"
 
 ```
+
+### Messages with quick responses
+
+You may want to offer quick, clickable responses to messages. Passing `quickResponses` to `<Message>` allows you to quickly add these custom actions. Quick actions are [PatternFly `<Label>` components](/components/label/) in a PatternFly `<LabelGroup>`, configurated to show up to 5 visible labels. This can be overriden by passing additional `<LabelGroup>` props to `quickResponseContainerProps`, or additional `<Label>` props to `quickResponses`.
```

Comment:
```suggestion
You can offer convenient, clickable responses to messages in the form of quick actions. Quick actions are [PatternFly labels](/components/label/) in a label group, configured to display up to 5 visible labels.

To add quick actions, pass `quickResponses` to `<Message>`. This can be overridden by passing additional `<LabelGroup>` props to `quickResponseContainerProps`, or additional `<Label>` props to `quickResponses`.
```
hm I think I would also change the last sentence slightly, but I don't know if I'm completely understanding it yet. When you say 'this can be overridden' - what do you mean by 'this'? Do you mean that you can add more than 5 labels, or customize the labels further?

I'm also curious about what `quickResponseContainerProps` does, since its not in the example code. Is it only used in certain scenarios? This might be extraneous info for this pr I'm just curious hah

Reply:
If you add more than 5, they start getting collapsed like https://staging.patternfly.org/components/label#label-group-with-overflow. (We're using the PatternFly component here.) The default in PatternFly is 3, then hide. We're showing up to 5, then hide, but you can pass quickResponseContainerProps to the LabelGroup to change that back or add any other PatternFly LabelGroup props. Do you think we should rename it to labelGroupContainerProps?

Reply:
I adjusted the text to match your suggestions, but let me know if you have additional changes!